README.md

```diff
@@ -210,6 +210,8 @@ A tag pattern can be:
 
 ### Display
 
+TODO
```

Comment:
* [ ] Rework the documentation
* [ ] Add a BREAKING CHANGE to the original commit as this is breaking existing configs